client/src/questions-n-answers/sub-components/mini-components/userAnswer.jsx

```diff
@@ -39,18 +46,20 @@ class UserAnswer extends React.Component {
       } else {
 
         this.setState({
+          fileName:this.state.fileName + 1,
```

Comment:
I worry for something like this where incrementing the state is very important that it might be worth it to pass prevState in to ensure it doesn't scuff. This is the docs that come to mind--> https://reactjs.org/docs/faq-state.html#what-is-the-difference-between-passing-an-object-or-a-function-in-setstate